esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -98,6 +98,21 @@ static IXmlSchema* createXmlSchema(const char* schema)
     return creator(schema);
 }
 
+static void setBndCfgServiceType(IPropertyTree* tree, const char* procName, IPropertyTree* bndCfg)
+{
+    const char* srvName = bndCfg->queryProp("@service");
+    if (isEmptyString(srvName))
+        return;
+
+    Owned<IPropertyTree> srvCfg = getServiceConfig(tree, srvName, procName);
+    if (!srvCfg)
+        return;
+
+    const char* srvType = srvCfg->queryProp("@type");
+    if (!isEmptyString(srvType))
+        bndCfg->addProp("@serviceType", srvType);
```

Comment:
trivial: setProp is more natural since you can't have more than one instance of an attribute.

Reply:
Will change